Sources/SwiftlySearch/SwiftlySearch.swift

```diff
@@ -86,11 +109,25 @@ fileprivate struct SearchBar: UIViewControllerRepresentable {
             }
         }
         
+        var hidesSearchBarWhenScrolling: Bool = true {
+            didSet {
+                self.parent?.navigationItem.hidesSearchBarWhenScrolling = hidesSearchBarWhenScrolling
+            }
+        }
```

Comment:
I don't see why this is necessary. Could you explain why this is needed in addition to line 127?

Reply:
I'm not 100% sure how SwiftUI manages `UIViewControllerRepresentable` objects and their view controller. But as I understand it, one `SearchBarWrapperController` object can be handled by multiple `SearchBar` objects during its lifecycle. Therefore `hidesSearchBarWhenScrolling` can be changed and has to be somehow updated. 
Probably not something a lot of people would do, but it's kind of SwiftUI's paradigm to be able to update anything.

Reply:
Yeah that makes sense. I also forgot I had a similar fix on `searchController` since the diff hid those lines.